lint.swift

```diff
@@ -0,0 +1,547 @@
+#!/usr/local/bin/swift-sh
+import AnyLint // @Flinesoft ~> 0.8.2
+
+try Lint.logSummaryAndExit(arguments: CommandLine.arguments) {
+	// MARK: - Variables
+	let swiftFiles: Regex = #"^Sources/.*\.swift$"#
+	let testFiles: Regex = #"^Tests/.*\.swift$"#
+  let readmeFile: Regex = #"README\.md"#
+  let changelogFile: Regex = #"^CHANGELOG\.md$"#
+
+	// MARK: - Checks
```

Comment:
Codacy found an issue: [Include a single vertical whitespace (empty line) before and none after MARK: comments.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)